index.js

```diff
@@ -725,6 +725,198 @@ PersonaClient.prototype._removeTokenFromCache = function (id, secret, callback)
 };
 
 /**
+ * Get scope information for a user
+ * @param guid
+ * @param token
+ * @param callback
+ */
+PersonaClient.prototype.getScopesForUser = function(guid, token, callback) {
+    try {
+        _.map([guid, token], function (arg) {
+            if (!_.isString(arg)) {
+                throw "guid and token are required strings";
+            }
+        });
+    } catch (e) {
+        callback(e,null);
+        return;
+    }
+
+    var _this = this;
+
+    var options = {
+            hostname: _this.config.persona_host,
+            port: _this.config.persona_port,
+            path: "/1/clients/" + guid,
+            method: 'GET',
+            headers: {
+                Authorization: "Bearer " + token,
+            },
+        },
+        req = _this.http.request(options, function (resp) {
+            if (resp.statusCode === 200) {
+                var str = '';
+
+                resp.on("data", function (chunk) {
+                    str += chunk;
+                });
+
+                resp.on("end", function () {
+                    var data;
+                    try {
+                        data = JSON.parse(str);
+                    } catch (e) {
+                        callback("Error parsing response from persona: " + str, null);
+                        return;
+                    }
+
+                    if (data) {
+                        if (data.error) {
+                            callback(data.error, null);
+                        } else {
+                            if (data.scope) {
+                                callback(null, data.scope);
+                            } else {
+                                callback("Scopes not found for Guid", null);
+                            }
+                        }
+                    } else {
+                        callback("Could not get Scopes for Guid", null);
+                    }
+                });
+            } else {
+                var err = "getScopesForUser failed with status code " + resp.statusCode;
+                _this.error(err);
```

Comment:
Can this error method be renamed to `logError`. It's not clear what it's doing with the error given that the callback is also using it alongside.